recipes/openmc-plotter/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "openmc-plotter" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/openmc-dev/plotter.git
+  git_rev: v{{ version }}
```

Comment:
This should be the git release, not the actual repo: https://github.com/openmc-dev/plotter/archive/refs/tags/v0.2.0.tar.gz